ui/packages/consul-ui/app/components/consul/auth-method/index.scss

```diff
@@ -24,9 +24,17 @@
         font-weight: $typo-weight-semibold;
         font-size: $typo-size-700;
       }
-      tbody td {
-        font-size: $typo-size-600;
-        color: $black;
+      tbody {
```

Comment:
@johncowen I thought about making this into a styling component but this is an overwrite of the already existing %table. Later on I will take a look at what can be refactored in %table to delete all this. For now, I'm just overwriting to get this working.

Reply:
Sounds great!